hikari/api/entity_factory.py

```diff
@@ -1619,3 +1620,52 @@ def deserialize_webhook(self, payload: data_binding.JSONObject) -> webhook_model
         hikari.webhooks.PartialWebhook
             The deserialized webhook object.
         """
+
+    ###################
+    # AUTO-MOD MODELS #
```

Comment:
Auto-Mod or AutoMod or Auto Mod?

Reply:
In this context Auto-Mod and Auto Mod are just the same thing and that's how hyphens be.
Also hyphens just aren't valid in python names so not sure what you're getting at.